### ISSUE TITLE
Link to view comments not working

### ISSUE THREAD

**Issue by fabianwenk** [jullrich/ISC-Bugs#83]
Links pointing to comments (like https://isc.sans.edu/forums/diary/ISC+Website+Redesign/28826/#comments) are not displaying the comments (not logged in).
Related: Not sure if this is intended or not, but I see no option to comment on a diary entry. Or is now an account needed for this?

**Comment by jc2it** [comment]
I agree with @fabianwenk that the comment section is either intentionally or unintentionally disabled. At least in the browsers I have currently tried. This is unfortunate as in the past I have learned much from the comments section. Here is an example of how important it can be. 
[https://isc.sans.edu/diary/Gems+in+the+ISC+Diary+Comments/17767](url)
Please do the community a favor and find a way to fix this. 
Thanks!

Edit: I have now tested in Firefox, Edge and an older IE (I mean you never know) install with the same results. Comments don't work. It is possible it works in chrome, but I rarely use it and don't want to install it only to test it.

**Comment by jc2it** [comment]
Please let your Handlers know that comments are either broken or disabled. 
Then we can avoid this frustration. 
"If you find a link shortener service where this doesn't work please let us know in the comment section?"
[https://isc.sans.edu/diary/Taking+Apart+URL+Shorteners/28980](url)

**Comment by jullrich** [comment]
yes. this is currently being worked on.
